CRM/Contribute/Form/ContributionPage/TabHeader.php

```diff
@@ -94,6 +94,10 @@ public static function process(&$form) {
       ] + $default,
     ];
 
+    if (!function_exists('tellafriend_config')) {
+      unset($tabs['friend']);
+    }
+
     $contribPageId = $form->getVar('_id');
     // Call tabset hook to add/remove custom tabs
     CRM_Utils_Hook::tabset('civicrm/admin/contribute', $tabs, ['contribution_page_id' => $contribPageId]);
```

Comment:
This hook can be used by the extension & the above code can be removed from core.

Reply:
Done. It moves the "Tell a Friend" to the last tab, but I think that's OK, since we're deprecating it.